metrics-operator/controllers/common/providers/dynatrace/dynatrace_dql.go

```diff
@@ -137,27 +203,42 @@ func (d *keptnDynatraceDQLProvider) ensureDTClientIsSetUp(ctx context.Context, p
 	return nil
 }
 
-func (d *keptnDynatraceDQLProvider) postDQL(ctx context.Context, query string) (*DynatraceDQLHandler, error) {
+func (d *keptnDynatraceDQLProvider) postDQL(ctx context.Context, metric metricsapi.KeptnMetric) ([]byte, int, error) {
 	d.log.V(10).Info("posting DQL")
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
 
-	values := url.Values{}
-	values.Add("query", query)
+	path := "/platform/storage/query/v0.7/query:execute"
+
+	payload := DQLRequest{
+		Query:                      metric.Spec.Query,
+		DefaultTimeframeStart:      time.Now().Format(time.RFC3339),
```

Comment:
by default we can leave the timeframe properties unset, otherwise we will get a timeframe with a length of 0

Reply:
Fixed.